nrf-hal-common/src/rtc.rs

```diff
@@ -203,6 +203,20 @@ where
         self.periph.counter.read().bits()
     }
 
+	/// Clear the Real Time Counter
+    pub fn clear_counter(&self) {
+        unsafe {
+            self.periph.tasks_clear.write(|w| w.bits(1));
+        }
+    }
+
+    /// Trigger the Real Time Counter to overflow, mainly for testing purposes
+    pub fn trigger_overflow(&self) {
```

Comment:
What happens if this is called while the timer is stopped? According to the infocenter article the counter would be set to 0xFFFFF0 (*not* 0xFFFFFF), so that wouldn't result in overflow - and even when it's running it wouldn't overflow immediately.

Maybe this should just document that it just sets the counter to 0xFFFFF0?

Reply:
Agreed. It makes the overflow occur sooner, for testing purposes. I'll change the documentation, should we stick with the name, as the task is named the same way?

Reply:
Yeah that should be fine